libflux/src/core/parser/tests.rs

```diff
@@ -1466,6 +1466,172 @@ fn test_parse_type_expression_regexp() {
     )
 }
 
+#[test]
+fn test_parse_record_only_properties() {
+    let mut p = Parser::new(r#"{a:int, b:uint}"#);
+    let parsed = p.parse_record();
+    let loc = Locator::new(&p.source[..]);
+    assert_eq!(
+        parsed,
+        Record {
+            base: BaseNode {
+                location: loc.get(1, 1, 1, 16),
+                ..BaseNode::default()
+            },
+            tvar: None,
+            properties: Some(vec![
+                PropertyType {
+                    base: BaseNode {
+                        location: loc.get(1, 4, 1, 7),
```

Comment:
Similar to my previous comment. The source location for this property should correspond to the entire property `a:int`, but right now it just refers to the location of the monotype `int`.